spec/factories/questions.rb

```diff
@@ -3,7 +3,7 @@
     title { Faker::StarWars.quote }
     content { Faker::Hipster.paragraph }
     views { rand(5) }
-    user
+    user factory: :user
```

Comment:
With this, when I create a question without passing a user object, it will build a user from the user factory and associate it to this question, and since a question belongs to a user, having a question without user association is not a good practice and will be disallowed in rails 5
`config.active_record.belongs_to_required_by_default = true`